docs/bin/plugin_formatter.py

```diff
@@ -132,6 +132,12 @@ def rst_xline(width, char="="):
     return char * width
 
 
+# NOTE: The existing Jinja2 'sequence' tests matches strings and dictionaries
+def test_list(value):
+    ''' Return true if objects is a list or tupple '''
+    return isinstance(value, Sequence)
```

Comment:
@dagwieers `str` is also `Sequence`. So it's either docstring or the check is misleading.
You might want to add `and not isinstance(value, str)`

Reply:
Yup, I know: https://github.com/ansible/ansible/pull/37517/files#diff-bc6bf5732a9c00dcb3cb182f4ce3b7c8R100

But that's no longer my problem now, is it :-)